### PR TITLE
normalize page font styles

### PR DESCRIPTION
**Pull request by brad-decker**
depends on #9693 

A piece of the larger #9074 that focuses only on page styles. This PR has the least broad changes to QA, in that the changes should be isolated to the modified pages. That could be a fallacy though so test to your heart's content ❤️

### CODE REVIEW

File: ui/app/pages/keychains/index.scss [hunk 1]
```diff
@@ -70,67 +70,41 @@
...
-    line-height: 23px;
-  }
-
   &__selector-label {
-    color: #1b344d;
-    font-size: 16px;
-  }
+    @include Paragraph;
 
-  &__dropdown {
```
Comment (darkwing) [inline ui/app/pages/keychains/index.scss:113]: Are this and many others simply not used?

**Review by darkwing — DISMISSED** [review]
One question but presume styles are being used.

File: ui/app/pages/keychains/index.scss (hunk 1 above)
Comment (brad-decker) [reply ui/app/pages/keychains/index.scss:113]: Yes, they seem to be older styles that are no longer used.

File: ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js [hunk 2]
```diff
@@ -156,16 +154,19 @@ export default class SendGasRow extends Component {
...
-      </SendRowWrapper>
+      <>
+        <SendRowWrapper
+          label={`${this.context.t('transactionFee')}:`}
+          showError={gasFeeError}
+          errorType="gasFee"
+        >
+          { this.renderContent() }
+        </SendRowWrapper>
+        {gasButtonGroupShown && <SendRowWrapper>{ this.renderAdvancedOptionsButton() }</SendRowWrapper>}
```
Comment (Gudahtt) [inline ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js:168]: Do you recall why these changes were made? I don't remember this part.

Comment (brad-decker) [reply ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js:168]: Phew, found [it](https://github.com/MetaMask/metamask-extension/pull/9074#discussion_r465990095) - slight improvement of how items are grouped to allow for better alignment

Comment (Gudahtt) [reply ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js:168]: Ah, thanks! Yes I see why you did this now - this is much better.

This does seem to be a change in logic though. Previously these buttons were also shown when `advancedInlineGasShown` was truthy. e.g.

```suggestion
        {
          gasButtonGroupShown || advancedInlineGasShown
            ? <SendRowWrapper>{ this.renderAdvancedOptionsButton() }</SendRowWrapper>
            : null
        }
```

(though this suggestion can't be accepted directly - the `advancedInlineGasShown` prop needs to be brought into scope.

Comment (brad-decker) [reply ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js:168]: ah thanks!

Comment (brad-decker) [reply ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js]: updated in last commit

**Review by Gudahtt — APPROVED** [review]
LGTM!
